int32 bbox_x1
int32 bbox_y1
int32 bbox_x2
int32 bbox_y2
float32 confidence
int32 class_id
string class_name